### PR TITLE
Add beginnings of alerts panel

### PR DESCRIPTION
**Pull request by allysonjp715**
Just shows static content for now (and not content appropriate for all cases). Will continue to update in future PRs.

Screen Shot 2020-08-17 at 4 23 28 PM

### CODE REVIEW

File: verification/curator-service/ui/src/components/Alerts.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    <Typography variant="body2">No alerts</Typography>
+                </div>
+            )}
+            {uploads.map((upload) => (
+                <div key={upload._id} className={classes.section}>
+                    <Typography variant="subtitle1">
+                        New source verification required
+                    </Typography>
+                    <Typography variant="body2">
+                        Please verify {upload.summary.numCreated} cases added
```
Comment (axmb) [inline verification/curator-service/ui/src/components/Alerts.tsx:69]: Would be good to display numUpdated -- cool to leave for later, but will be a little goofy in a pure update for now.

File: verification/curator-service/ui/src/components/Alerts.test.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+afterEach(() => {
+    jest.clearAllMocks();
+});
+
+it('loads and displays alerts', async () => {
+    const uploads = {
+        uploads: [
+            {
+                _id: '5ef8e943dfe6e00030892d58',
+                status: 'IN_PROGRESS',
```
Comment (axmb) [inline verification/curator-service/ui/src/components/Alerts.test.tsx:18]: Do we want to filter out IN_PROGRESS? Displaying only SUCCESS/ERROR.

File: verification/curator-service/ui/src/components/Alerts.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    _id: string;
+    status: string;
+    summary: UploadSummary;
+    created: Date;
+}
+
+interface UploadSummary {
+    numCreated: number;
+    numUpdated: number;
+    error: string;
```
[the remark this answers is not in the document]
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Alerts.tsx:27]: Done

File: verification/curator-service/ui/src/components/Alerts.tsx (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Alerts.tsx]: Done

File: verification/curator-service/ui/src/components/Alerts.test.tsx (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/Alerts.test.tsx:18]: Yes, we should also filter for actioned uploads. Currently thinking that's best to do in the API endpoint, so will leave for a future PR.

**Pull request merged**